django_migration_vis/management/commands/visualizemigrations.py

```diff
@@ -32,7 +39,26 @@ def _create_digraph(self):
             self._add_dependencies(node)
 
     @staticmethod
-    def _style_label(tupled_node):
+    def _censor(text):
+        res = []
+        for c in text:
+            if c not in '0123456789_':
+                c = chr(random.randint(ord('a'), ord('z')))
```

Comment:
Is it your intention to keep the lengths of node names or was it for convenience? Do you want to recognize what they were called before? what level of anonymity are you considering?

Reply:
Mostly convenience. There are no strict rules about how obfuscated results need to be.